ECoG_MAE/downstream_tasks/encoding/load_signal.py

```diff
@@ -0,0 +1,136 @@
+# Here we want to load the signal for the grid electrodes and organize it in an 8*8 matrix. We also want to perform filtering here - we might need to think about how to do it
+# such that it is the same as the filtering we perform for model training - since there we do it using mne, which we can't here, since the preprocessed signal here are mat files.
```

Comment:
We can get the raw data and decide how we want to save (and process?) it. Do you have access to the raw edf file with this patient's podcast data?

Reply:
I don't think I do currently. The way it's set up now is just using the 64 grid electrode data which Leo put on huggingface. If I had access to the edf file I could just use the existing data loader as well.

Reply:
Great, maybe take a look at the current main branch in the encoding repo, it's been updated with more efficient regression (using himalaya)